### PR TITLE
CVE-2012-3443

### PR DESCRIPTION
**Pull request by Liam-Miller-SE**

### CODE REVIEW

File: cves/CVE-2012-3443.yml [hunk 1]
```diff
@@ -49,7 +49,10 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: When uploading an image to the django.forms.ImageField object,
+  the image is decompressed to the full resolution when being drawn to the
+  image box. This takes up processing time and a potential attacker could
+  upload many heavily compressed, large images to slow the system.
```
Comment (GORosin) [inline cves/CVE-2012-3443.yml:56]: Do you know why the image is decompressed?

File: cves/CVE-2012-3443.yml [hunk 2]
```diff
@@ -264,7 +269,8 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: This is less of a mistake and more of an oversight, there is code to ensure
+    images displayed are legitimate, but the size and compression amount are not enforced
```
Comment (GORosin) [inline cves/CVE-2012-3443.yml:274]: You can also limit the size of images that can be uploaded and the amount of compression. It wasn't in this case, which feels like a design flaw to me.

File: cves/CVE-2012-3443.yml [hunk 3]
```diff
@@ -277,10 +283,10 @@ CWE_instructions: |
...
   apply here, then choose the best one and mention the others in CWE_note.
-CWE:
+CWE: 409
 CWE_note:
 nickname_instructions: |
   A catchy name for this vulnerability that would draw attention it. If the
   report mentions a nickname, use that. Must be under 30 characters.
   Optional.
-nickname:
+nickname: Compressed Image Bomb
```
Comment (GORosin) [inline cves/CVE-2012-3443.yml:294]: Upvotes: 3

File: cves/CVE-2012-3443.yml [hunk 4]
```diff
@@ -101,11 +104,11 @@ unit_tested:
 
     For the fix_answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
-  code:
+  code: false
   code_answer:
```
Comment (mesternefeld) [inline cves/CVE-2012-3443.yml:109]: code_answer and fix_answer should be not null

File: cves/CVE-2012-3443.yml [hunk 5]
```diff
@@ -136,8 +139,8 @@ autodiscoverable:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  answer_note:
-  answer:
+  answer_note: the original vulnurability was found by an automated tool
```
Comment (mesternefeld) [inline cves/CVE-2012-3443.yml:143]: Interesting. Can you specify which one

**Pull request closed without merging**